src/Validation/Validation.php

```diff
@@ -1021,6 +1023,74 @@ public static function uploadedFile($file, array $options = [])
     }
 
     /**
+     * Validates a geographic coordinate.
+     *
+     * Supported formats are right now:
+     *
+     * - <latitude>, <longitude> Example: -25.274398, 133.775136
+     *
+     * ### Options
+     *
+     * - `type` - A string of the coordinate format, right now only `latLong`.
```

Comment:
We can have separate method for those. Don't see need to have the ability to check for all formats using same method. 

I also don't particularly like the other option name `latLong`.  It could be `format` or `pattern`.